planner/core/point_get_plan.go

```diff
@@ -749,6 +764,16 @@ func tryPointGetPlan(ctx sessionctx.Context, selStmt *ast.SelectStmt) *PointGetP
 	return nil
 }
 
+func partitionNameInSet(name model.CIStr, pnames []model.CIStr) bool {
```

Comment:
Is there a performance issue when the partition count is large, For example: 65535.

Reply:
This name slice is not all the partitions of a table.
It's just the selected ones:
```
select * from t partition (p0,p1,p2) ...
```

A sane person would not write a SQL like this:

```
select * from t partition (p0,p1,p2....p65535) ...
```